libcaliptra/inc/caliptra_enums.h

```diff
@@ -31,3 +31,44 @@ enum toc_entry_id {
     MAX     = 0xFFFFFFFF,
 };
 
+// The below enums are placeholders to set up the baseline
+// required for communication of DPE commands to Caliptra
+// firmware.
+
+enum dpe_commands {
+    DPE_GET_PROFILE        = 0x1,
+    DPE_INITIALIZE_CONTEXT = 0x7,
+    DPE_DERIVE_CHILD       = 0x8,
+    DPE_CERTIFY_KEY        = 0x9,
+    DPE_SIGN               = 0xA,
+    DPE_ROTATE_CTX_HANDLE  = 0xE,
```

Comment:
Not for this patch, can this eventually be converted from RUST and taken from the DPE Library ?

Reply:
I have attempted to run cbindgen on the DPE library and it consistently fails when attempting to process some of the impl statements for the structures. If there's a way to have it convert only structs and enums, that would be extremely helpful and I would replace everything with them.

Reply:
Let me know if there are any changes we can make to the struct/enum definitions to make this easier

Reply:
Unfortunately I wouldn't know what changes to make as I'm still ramping on Rust and most of my time thus far has me still working in the C world, and my time allocation has shifted away from Caliptra.

Reply:
Ack. I can take some time to look into cleaning this up in a follow-up CL. Could you open an issue for this?

Reply:
https://github.com/chipsalliance/caliptra-sw/issues/752